From c681646e7bcdebd21121fbb44f1282d9a0ef7f3f Mon Sep 17 00:00:00 2001
From: Wu Guoxing <b02248@freescale.com>
Date: Thu, 14 Aug 2008 16:15:06 +0800
Subject: [PATCH] ENGR00083952 The first frame will show one strap on the top

The first frame will show one strap on the top,
this is because the sensor need some time
to setup, so, add some delay.

Signed-off-by: Guoxing Wu <b02248@freescale.com>
(cherry picked from commit 18902a5dc2890ce4a47995a2f8fd67d5f9a9021b)
---
 drivers/media/video/mxc/capture/ov2640.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/media/video/mxc/capture/ov2640.c b/drivers/media/video/mxc/capture/ov2640.c
index f768291..9257ab5 100644
--- a/drivers/media/video/mxc/capture/ov2640.c
+++ b/drivers/media/video/mxc/capture/ov2640.c
@@ -378,6 +378,8 @@ static sensor_interface *ov2640_config(int *frame_rate, int high_quality)
 	else
 		ov2640_init_mode(ov2640_mode_800_600);
 
+	msleep(300);
+
 	return interface_param;
 }
 
-- 
1.5.4.4

